sdk/typescript/runtime/main.go

```diff
@@ -93,6 +93,7 @@ func (t *TypescriptSdk) ModuleRuntime(ctx context.Context, modSource *ModuleSour
 		return ctr.
 			// Install dependencies
 			WithExec([]string{"yarn", "install"}).
+			WithExec([]string{"npm", "pkg", "set", "type=module"}).
```

Comment:
Why is this necessary?

Reply:
Because of `graphql-request`, it dropped the support for CommonJS, in order to be as smooth as possible with the transition, I set the package.json type to module in the runtime so an user who didn't add this entry to its package will see it working.